packages/react/src/components/header/Header.stories.tsx

```diff
@@ -191,6 +191,81 @@ export const Minimal = (args) => {
   );
 };
 
+export const MinimalWithLocalization = (args) => {
+  const [lang, setLang] = useState<string>('fi');
+
+  const translations = {
+    en: {
+      'header-title': 'Helsinki city',
+      'header-aria-label': 'Helsinki city',
+      'header-login': 'Login',
+      'header-search': 'Search',
+      'header-menu-title': 'Other languages',
+    },
+    fi: {
+      'header-title': 'Helsingin kaupunki',
+      'header-aria-label': 'Helsingin kaupunki',
+      'header-login': 'Kirjaudu',
+      'header-search': 'Haku',
+      'header-menu-title': 'Tietoa muilla kielillä',
+    },
+    sv: {
+      'header-title': 'Helsingfors Stad',
+      'header-aria-label': 'Helsingfors Stad',
+      'header-login': 'Logga in',
+      'header-search': 'Sök',
+      'header-menu-title': 'Andra språk',
+    },
+  };
+
+  const languageChangedAction2 = (lg: string) => setLang(lg);
+
+  return (
+    <Header {...args} onDidChangeLanguage={languageChangedAction2}>
+      <Header.ActionBar
+        title={translations[lang]['header-title']}
+        titleAriaLabel={translations[lang]['header-aria-label']}
+        titleUrl="https://hel.fi"
+        titleStyle={TitleStyleType.black}
+      >
+        <Header.NavigationLanguageSelector languages={languages}>
+          <h3>{translations[lang]['header-menu-title']}</h3>
+          <Link external href="www.example.com">
+            Selkosuomi
+          </Link>
+          <Link external href="www.example.com">
+            Viittomakieli
+          </Link>
+        </Header.NavigationLanguageSelector>
+
+        <Header.ActionBarItem
+          fullWidth
+          label={translations[lang]['header-search']}
+          icon={IconSearch}
+          id="action-bar-search"
+        >
+          <Header.NavigationSearch onChange={searchChangeAction} onSubmit={searchSubmitAction} label="Haku" />
```

Comment:
The `label="Haku"` should be changed to `label={translations[lang]['header-search']}`